fuzz/fuzz_targets/c_rust_merkle.rs

```diff
@@ -8,9 +8,22 @@ use simplicity::jet::Elements;
 use simplicity::{BitIter, RedeemNode};
 
 fn do_test(data: &[u8]) {
-    let c_result = run_program(data, TestUpTo::CheckOneOne);
+    if data.len() < 4 {
+        return;
+    }
+    let prog_len = (usize::from(data[0]) << 24)
+        + (usize::from(data[1]) << 16)
+        + (usize::from(data[2]) << 8)
+        + usize::from(data[3]);
```

Comment:
0b5c69f638177d5a0dd3916b0681d18f2e72fddd: Does this decode the program length from the Simplicity encoding? If so, let's mention this in a comment.

Reply:
Sure, I'll add a comment. Actually I think I can improve it.